CDP4ServicesDal/CDP4ServicesDal.csproj

```diff
@@ -39,14 +40,10 @@
     <None Include="..\README.md" Pack="true" PackagePath="\" />
   </ItemGroup>
 
-  <ItemGroup>
-      <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="6.0.0" />
```

Comment:
surprised to see that this can be removed, @lxatstariongroup  can you help me identify why this is no longer a dependency?

Reply:
Because another library was added. It is all about System.Net.Http.Formatting if I understood correctly. Will take a look and dive in deeper.

Reply:
Because @samatstariongroup asked to try to remove as much references to NewtonSoft, Microsoft.AspNet.WebApi.Client was removed, as it as  reference to NewtonSoft. 
It was replaced by **<PackageReference Include="System.Net.Http.Formatting.Extension" Version="5.2.3" />**.
Unfortunately System.Net.Http.Formatting.Extension was discontinued after versio 5.2.3, so that is unusable.
Microsoft.AspNet.WebApi.Client should be used for this as the IME also uses it.